exec/java-exec/src/main/java/org/apache/drill/exec/store/dfs/easy/EasyGroupScan.java

```diff
@@ -230,6 +230,16 @@ public void modifyFileSelection(FileSelection selection) {
     this.selection = selection;
   }
 
+  @Override
+  protected boolean supportsFileImplicitColumns() {
+    return true;
+  }
```

Comment:
This same info is passed to the EVF-based mechanism. Should this instead be specified on the format plugin and used both here and in the EVF builder so we get consistent results? Otherwise, how will folks like @cgivre know to specify the same thing twice?